src/plugins/discover/public/application/embeddable/search_embeddable.ts

```diff
@@ -353,6 +358,7 @@ export class SearchEmbeddable
       this.searchScope.$apply(() => {
         this.searchScope!.hits = resp.hits.hits;
         this.searchScope!.totalHitCount = resp.hits.total;
+        this.searchScope!.isLoading = false;
       });
     } catch (error) {
       this.updateOutput({ loading: false, error });
```

Comment:
Should we update `isLoading` here, on error as well? (Otherwise it might be stuck in the loading status forever?)

Reply:
good catch! will do! only diamonds should last forever!